desparchado/settings/dev.py

```
@@ -24,6 +21,10 @@ def show_debug_toolbar(request):
     'SHOW_TOOLBAR_CALLBACK': show_debug_toolbar,
 }
 
+# Indicates whether to serve assets via the ViteJS development server
+# or from compiled production assets
+DJANGO_VITE["default"]["dev_mode"] = True
```

Ruff (0.8.2)

`DJANGO_VITE` may be undefined, or defined from star imports

(F405)